testing/tests/DevExpress.ui.widgets.dataGrid/fixedColumns.tests.js

```diff
@@ -1850,19 +1850,21 @@ QUnit.testInActiveWindow("Scrolling to focused cell when it is fixed", function(
     $fixedTable = $testElement.find(".dx-datagrid-rowsview").children(".dx-datagrid-content-fixed").find("table");
     $cell = $fixedTable.find("tbody > tr:not(.dx-freespace-row)").last().children().first();
 
-    var scrollChanged = function(e) {
-        that.rowsView.scrollChanged.remove(scrollChanged);
-        scrollTop = -parseFloat($fixedTable.css("top"));
-        assert.ok(scrollTop > 500, "scroll top of the fixed table");
-        assert.ok(that.rowsView._scrollTop > 500, "scroll top of the main table");
-        assert.equal(scrollTop, that.rowsView._scrollTop, "scroll top of the fixed table equal scroll top of the main table");
-        done();
-    };
+    setTimeout(function() {
+        var scrollChanged = function(e) {
```

Comment:
@Alyar666  `scrollChanged` could be kept outside. Looks like only `focus()` needs to be delayed.

@vconst what do you think about the use of `setTimeout` here?